pkg/rpm/salt-minion

```diff
@@ -70,7 +68,8 @@ _su_cmd() {
 
 _get_pid() {
     netstat $NS_NOTRIM -ap --protocol=unix 2>$ERROR_TO_DEVNULL \
-        | sed -r  -e "\|\s${SOCK_DIR}/minion_event_${MINION_ID_HASH}_pub\.ipc$|"'!d; s|/.*||; s/.*\s//;'
+        | sed -r -e "\|\s${SOCK_DIR}/minion_event_${MINION_ID_HASH}_pub\.ipc$|"'!d; s|/.*||; s/.*\s//;' \
+        | uniq
```

Comment:
@terminalmage This should work as expected, returning unique PIDs of all running minions.